### DISCUSSION TITLE
Playbook rules still generating alerts after set to inactive or disabled

### DISCUSSION THREAD

**Discussion by 0nnyx** [Security-Onion-Solutions/securityonion#1951] [Unsupported Versions] [answered]
After initially activating critical & high playbook rules, I wanted to filter some generating false positives like Renamed Powershell & Highly Relevant Renamed Binary. 
Setting these 2 rules to inactive for a day, then to disabled the next day, I was still getting alerts for both. Also tried a reboot without success.
What Is the proper method to disable playbook rules ?

**Comment by defensivedepth** [comment, marked as answer]
We recently fixed a bug that was causing inactive Plays to continue to alert - that fix will be available in our next release. 

For now, you can delete the yaml files in `/opt/so/rules/elastalert/playbook/` and then run `sudo so-playbook-sync` - this will manually turn off the alerting for Plays that are currently inactive.
